controllers/csi/driver/server.go

```diff
@@ -0,0 +1,327 @@
+/*
```

Comment:
Is there a reason we have several methods which seem to be auto-generated but unimplemented?

Reply:
We need to fulfill an Go interface, even if we don't do anything. They are used for normal CSI drivers with PersistentVolumes. Our CSI driver supports only ephemeral volumes, so most of them aren't needed.

Reply:
OK